extensions/amp-date-countdown/1.0/README.md

```diff
@@ -0,0 +1,318 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: Displays a countdown sequence to a specified date.
+experimental: true
+bento: true
```

Comment:
this can be removed

Reply:
should I remove both the `experimental` and the `bento` properties? it looks like the other README should be updated as well